modules/dataplex-datascan/README.md

```diff
@@ -137,6 +137,19 @@ module "dataplex-datascan" {
         table_condition_expectation = {
           sql_expression = "COUNT(*) > 0"
         }
+      },
+      {
+        dimension = "VALIDITY"
+        sql_assertion = {
+          sql_statement = <<EOT
+            SELECT
+              city_asset_number, council_district
+            FROM $${data()}
+            WHERE city_asset_number IS NOT NULL
+            GROUP BY 1,2
+            HAVING COUNT(*) > 1
+          EOT
+        }
```

Comment:
Can you add this to inventory file? (datascan_dq.yaml)?

Reply:
Will do. Thank you.